portfolio/src/main/java/com/google/sps/data/CalendarEvent.java

```diff
@@ -0,0 +1,34 @@
+package com.google.sps.data;
+
+/** Models a calendar event that the user creates. */
```

Comment:
Do you plan to use this data model for imported calendar events? In that case, this comment may be too restrictive. Consider updating the comment to,

Data model for a calendar event. It can be used for both creation and import flows.

Reply:
Updated. The class should be used for both created and imported events.